README.md

```diff
@@ -246,50 +269,51 @@ At this point, cluster1 has an identity in cluster2, credentials to be authentic
 
 ClusterTargets and Targets are Kubernetes [custom resources](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) installed with multicluster-scheduler:
 
-- A cluster-scoped ClusterTarget associates a name (`metadata.name`) with a reference to a kubeconfig secret (name and namespace; can be in any namespace but it makes sense to store these in the same namespace as multicluster-scheduler, e.g., `admiralty`, as we did). Multicluster-scheduler will use the kubeconfig to interact with resources in the target cluster at the cluster level, so it must be authorized to do so there (i.e., there must be a ClusterRoleBinding between the `multicluster-scheduler-source` ClusterRole and cluster1's identity).
+- A cluster-scoped ClusterTarget references a kubeconfig secret (name and namespace; can be in any namespace but it makes sense to store these in the same namespace as multicluster-scheduler, e.g., `admiralty`). Multicluster-scheduler will use the kubeconfig to interact with resources in the target cluster at the cluster scope, so it must be authorized to do so there (i.e., cluster2 must have a ClusterSource for cluster1's identity, or a ClusterRoleBinding between the `multicluster-scheduler-source` ClusterRole and cluster1's identity).
 
-- A namespaced Target associates a name with a reference to a kubeconfig secret (name only, in the same namespace). Multicluster-scheduler will use the kubeconfig to interact with resources in the target cluster in the eponymous namespace (except to view the cluster-scoped ClusterSummary, as explained above), so it must be authorized to do so there (i.e., there must be a RoleBinding between the `multicluster-scheduler-source` ClusterRole and cluster1's identity in that namespace).
+- A namespaced Target references a kubeconfig secret in the same namespace. Multicluster-scheduler will use the kubeconfig to interact with resources in the target cluster in that namespace (except to view the cluster-scoped ClusterSummary, as explained above), so it must be authorized to do so there (i.e., cluster2 must have a Source for cluster1's identity in that namespace, or a RoleBinding between the `multicluster-scheduler-source` ClusterRole and cluster1's identity).
 
-ClusterTargets and Targets can also target their local cluster, e.g., for cloud bursting. In that case, they don't reference a kubeconfig secret, but specify `spec.self=true` instead. In our case, apply the following two ClusterTargets in cluster1:
+ClusterTargets and Targets can also target their local cluster, e.g., for cloud bursting. In that case, they don't reference a kubeconfig secret, but specify `spec.self=true` instead. In our case, apply the following two Targets in cluster1:
```

Comment:
Can we keep the ClusterTarget example?
It is still supported, right?

If you want to discourage that deployment model, maybe put it as an optional section?

Reply:
ClusterTarget is supported. Following the principle of least privilege, I switched to namespaced Targets for the getting started guide, but you're right, I should include a ClusterTarget example in a collapsible section, to mirror the ClusterSource example above.